### DISCUSSION TITLE
Custom CSS no longer working since release 9.1.7

### DISCUSSION THREAD

**Discussion by stevejr** [squidfunk/mkdocs-material#5424] [General]
Hi,

We have a small bit of custom CSS that we use for our site which has suddenly stopped working as of release 9.1.7

The CSS we have in the extra.css file is as follows:

```
:root {
  --md-primary-fg-color:        #3d90ce;
  --md-accent-fg-color:         #3d90ce;
}

:root > * {
  /* Footer */
  --md-footer-bg-color:         #e8e8e8;
  --md-footer-bg-color--dark:   #cccdcf;
  --md-footer-fg-color:         #3d90ce;
  --md-footer-fg-color--light:  #fff;

  /* Background */
  --md-default-bg-color:        #f1f1f1;
}

[data-md-color-scheme="slate"] {
  /* Footer */
  --md-footer-bg-color:         #2e3136;
  --md-footer-bg-color--dark:   #262b2f;
  --md-footer-fg-color:         #3d90ce;
  --md-footer-fg-color--light:  #fff;

  /* Background */
  --md-default-bg-color:        #1e2124;
}
```

When we build the site with release 9.1.7+ none of the custom colours are used - instead the default indigo colour is used. Has anyone else seen this issue?

Thanks.

**Comment by squidfunk** [comment]
Thanks for reporting. We made colors more explicit in 9.1.7 because there were some case where the wrong colors were applied. See #5206 for further information. Could you try to set the colors you want to overwrite in `mkdocs.yml` explicitly to `custom` and see if the issue persists? Specifically:

``` yaml
theme:
  palette:
    - scheme: ...
      primary: custom # <-- set to custom, so defaults are not applied
      accent: custom
```

**Reply by stevejr** [reply]
Thanks Martin, will give that a go.

Question - where you say set it to custom do you mean the actual word custom or the custom colour hex value?

Are Primary and Accent the only values to set in the mkdocs.yaml or or there others?

**Reply by squidfunk** [reply]
The actual word `custom`. Specific color values cannot be set through `mkdocs.yml` directly.

**Reply by stevejr** [reply]
@squidfunk - the above works! Thank you so much.

Would it be possible to update the docs to make this setup clear (I just checked and couldn't see anything in the docs that says we should set `primary: custom` when using a custom colour setup.

**Reply by squidfunk** [reply]
Yes, of course. I first wanted to check with you if this fixes the issue but we should definitely update the docs. Could you create a new documentation issue, so that we don't forget to update it?

**Reply by stevejr** [reply]
Done - https://github.com/squidfunk/mkdocs-material/issues/5427
